trustwallet/wallet-core
# New ‘newevmchain’ script, subset of ‘newcoin’

## Description

Added new `newevmchain` script, a subset of `newcoin`, to be used in the EVM chain case. It skips some changes/additions that are not needed in this use case.
Fixes #2791.

## How to test

Add a new Etehreum chain to `registry.json`, invoke `codegen/bin/newevmchain <id>`, observe changed/added files.

## Types of changes





* New feature (non-breaking change which adds functionality)



## Checklist




If you're adding a new blockchain
